rpm/android-tools-hadk.spec

```diff
@@ -54,7 +54,8 @@ make -f %{SOURCE3} -C core/mkbootimg
 %install
 rm -rf %{buildroot}
 install -D -m 755  core/fastboot/fastboot %{buildroot}%{_bindir}/fastboot
-install -D -m 755  mer-android-chroot %{buildroot}%{_bindir}/ubu-chroot
+sed -e 's/@VERSION@/%{version}/' mer-android-chroot > %{buildroot}%{_bindir}/ubu-chroot
```

Comment:
This line and one below should belong to the "Add -V option..." commit (sorry I can't select both lines through mobile browser:)

New commit has nitpick typo -> behaviour.

Reply:
Still that typo in the last comment